addons/payment_xendit/models/payment_transaction.py

```diff
@@ -122,7 +122,7 @@ def _process_notification_data(self, notification_data):
         self.ensure_one()
 
         super()._process_notification_data(notification_data)
-        if self.provider_code != 'xendit':
+        if self.provider_code != 'xendit' and self.payment_method != 'card':
```

Comment:
runbot is red :eyes: 
Why did you add this? We need to process all transactions regardless of the payment method.
```suggestion
        if self.provider_code != 'xendit':
```